lib/flowViewport/API/perViewportSceneIndicesData/fvpViewportInformationAndSceneIndicesPerViewportDataManager.cpp

```diff
@@ -174,6 +174,7 @@ void ViewportInformationAndSceneIndicesPerViewportDataManager::DisableIsolateSel
 )
 {
     _isolateSelection[viewportId] = nullptr;
+    _isolateSelectSceneIndex->SetViewport(viewportId, nullptr);
```

Comment:
Fixed latent bug that was being compensated for in renderOverride.cpp.  The Flow Viewport Toolkit should not rely on clients to manage the isolate select scene index properly when it can do so itself.